charts/oidc-proxy/ci/testing-values.yaml

```diff
@@ -5,6 +5,7 @@ env:
   PROXY_URL3: http://test-datarepo-api.integration-temp:8080/ga4gh
   PROXY_PATH3: /ga4gh
   LOG_LEVEL: debug
+  CORS_REGEX: http(s)?://(www\\.)?(.+\\.broadinstitute\\.org|.+\\envs-terra\\.bio|.+\\.terra\\.bio|.+\\.appspot\\.com|localhost)$
```

Comment:
Note: this won't match `localhost` because it expects there to be https in front of it.  I'm also wondering if we even need localhost since in that case we're not going through the apache proxy

Reply:
I think `http(s)?` makes the `s` optional. But you may be right, that this is unnecessary since localhost isn't used in a helm deployed environment.

Reply:
Oh gosh, you're totally right, duh.  I do wonder if we would need the port numbers (3000 and 8080)